libvast/fbs/type.fbs

```diff
@@ -89,6 +89,7 @@ union Type {
   duration_type: DurationType,
   time_type: TimeType,
   string_type: StringType,
+  // NOTE: The PatternType was removed with VAST v3.0.
```

Comment:
Can we mark the field as `(deprecated)` then?

Reply:
That doesn't work on union members, and setting a manual `(id: 9)` for the field after it to skip one index only works for a FlatBuffers version that is too recent for Debian.